tools/build_android/android_build.sh

```diff
@@ -21,6 +25,6 @@ if [ "$ARCH" == "armeabi-v7a" ]; then
     export TOOLCHAIN_NAME='android-ndk-r16b-api-21-x86-clang-libcxx'
 fi
 
-cmake -DCMAKE_BUILD_TYPE:STRING=Release -DTARGET_JNI=ON -DCMAKE_TOOLCHAIN_FILE=${POLLY_ROOT}/${TOOLCHAIN_NAME}.cmake ../lib-ledger-core
+cmake . -DCMAKE_BUILD_TYPE:STRING=Release -DTARGET_JNI=ON -DCMAKE_TOOLCHAIN_FILE=${POLLY_ROOT}/${TOOLCHAIN_NAME}.cmake
```

Comment:
I think that this one should be `libcore_path`, since we are supposed to launch it from build dir, or you will have to change the line below since it's building current from directory